src/test_lib_json/main.cpp

```diff
@@ -1464,8 +1464,7 @@ JSONTEST_FIXTURE_LOCAL(ValueTest, nonIntegers) {
 void ValueTest::checkConstMemberCount(const Json::Value& value,
                                       unsigned int expectedCount) {
   unsigned int count = 0;
-  Json::Value::const_iterator itEnd = value.end();
-  for (Json::Value::const_iterator it = value.begin(); it != itEnd; ++it) {
+  for ([[maybe_unused]] auto it : value) {
```

Comment:
Anyone got a solution for this?

[[maybe_unused]] is ignored in VS2015. VS2017 demands a c++17 flag (as this is a c++17 feature).

GCC seems to work.

__attribute__((unused)) works in GCC but not MSVC.